controls/1_3_filesystem_integrity_checking.rb

```diff
@@ -48,14 +48,14 @@
   describe.one do
     %w(/var/spool/cron/crontabs/root /var/spool/cron/root /etc/crontab).each do |f|
       describe file(f) do
-        its(:content) { should match(/aide --check/) }
+        its(:content) { should match(/aide (--check|-C)/) }
       end
     end
 
     %w(cron.d cron.hourly cron.daily cron.weekly cron.monthly).each do |f|
       command("find /etc/#{f} -type f").stdout.split.each do |entry|
         describe file(entry) do
-          its(:content) { should match(/aide --check/) }
+          its(:content) { should match(/aide (--check|-C)/) }
```

Comment:
would using the  [crontab resource](https://docs.chef.io/inspec/resources/crontab/) make this any cleaner?

Reply:
I don't think there is a easy way to query all crontabs for all users easily with the crontab resource. After looking at the docs, I think this is the simplest way to go